x-pack/test/observability_ai_assistant_functional/tests/conversations/index.spec.ts

```diff
@@ -287,6 +288,17 @@ export default function ApiTest({ getService, getPageObjects }: FtrProviderConte
                   expect(response.body.conversations[0].messages.length).to.eql(9);
                 });
               });
+
+              describe('and choosing to send feedback', () => {
+                before(async () => {
+                  await telemetry.setOptIn(true);
+                });
```

Comment:
Telemetry was disabled when running this test. That's why the event was not captured below.

I noticed you had it on a `before` at the top. However, this helper only opts-in _in-memory_ and somewhere in the tests, we refresh the page (losing the `setOptIn` request).